gthockey-iOS/gthockey-iOS/ScheduleFlow/Game.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Game.swift
+//  gthockey-iOS
+//
+//  Created by Dylan Mace on 9/27/19.
+//  Copyright © 2019 Caleb Rudnicki. All rights reserved.
+//
+
+import Foundation
+
+class Game {
+    private var id: Int
+    private var dateTime: String
+    private var opponentName: String
+    private var rinkName: String
+    private var venue: String
+    private var isReported: Bool
+    private var shortResult: String
+    private var gtScore: Int
+    private var opponentScore: Int
+    
+    init(id_: Int, dt: String, on: String, rn: String, v: String, ir: Bool, sr: String, gs:Int, os: Int) {
```

Comment:
Lets get the parameter names to be the same as the attribute names. And then when you set do something like `self.rinkName = rinkName`. It's best to keep names as descriptive as possible.